[2024-06-19T10:30:31.518+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-19T10:30:31.539+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Backup_Database.backup_task manual__2024-06-19T05:00:30.343958+00:00 [queued]>
[2024-06-19T10:30:31.552+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Backup_Database.backup_task manual__2024-06-19T05:00:30.343958+00:00 [queued]>
[2024-06-19T10:30:31.552+0530] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-19T10:30:31.577+0530] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): backup_task> on 2024-06-19 05:00:30.343958+00:00
[2024-06-19T10:30:31.579+0530] {standard_task_runner.py:63} INFO - Started process 4846 to run task
[2024-06-19T10:30:31.584+0530] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Backup_Database', 'backup_task', 'manual__2024-06-19T05:00:30.343958+00:00', '--job-id', '136', '--raw', '--subdir', 'DAGS_FOLDER/basic.py', '--cfg-path', '/tmp/tmpq564ihpg']
[2024-06-19T10:30:31.586+0530] {standard_task_runner.py:91} INFO - Job 136: Subtask backup_task
[2024-06-19T10:30:31.663+0530] {task_command.py:426} INFO - Running <TaskInstance: Backup_Database.backup_task manual__2024-06-19T05:00:30.343958+00:00 [running]> on host FN-21052948.
[2024-06-19T10:30:31.750+0530] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sarika' AIRFLOW_CTX_DAG_ID='Backup_Database' AIRFLOW_CTX_TASK_ID='backup_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-19T05:00:30.343958+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-19T05:00:30.343958+00:00'
[2024-06-19T10:30:31.751+0530] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-19T10:30:31.762+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-19T10:30:31.762+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/kiit/airflow/bashScripts/backup.sh ']
[2024-06-19T10:30:31.766+0530] {subprocess.py:86} INFO - Output:
[2024-06-19T10:30:31.776+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:30:31.782+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:30:31.799+0530] {subprocess.py:93} INFO - mysqldump: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:30:32.087+0530] {subprocess.py:93} INFO - Backup completed in 1 seconds
[2024-06-19T10:30:32.087+0530] {subprocess.py:93} INFO - Backup Success in location: /home/kiit/airflow/backup | Required Space: 0GB, Available Space: 245.31GB
[2024-06-19T10:30:32.088+0530] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-19T10:30:32.088+0530] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-19T10:30:32.121+0530] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Backup_Database, task_id=backup_task, run_id=manual__2024-06-19T05:00:30.343958+00:00, execution_date=20240619T050030, start_date=20240619T050031, end_date=20240619T050032
[2024-06-19T10:30:32.158+0530] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-19T10:30:32.180+0530] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-19T10:30:32.188+0530] {local_task_job_runner.py:222} INFO - ::endgroup::
